playbooks/azure/openshift-cluster/group_vars/all/generic.yml

```diff
@@ -0,0 +1,3 @@
+---
+openshift_azure_release: 3.10
```

Comment:
I thought we should be passing this from the job config?

Reply:
and I thought we will not do this and will maintain logic in branches in openshift-ansible? :confused:

Reply:
Right, master is a special case though and we need to specify the version anyway somewhere, is thta correct? I would avoid hardcoding this here unless there is precedent.

Reply:
Step back. You suggest to hardcode in release branches and pass it to master? I think there is no point in doing one or other way for different branches. We either pass this for all branches (master will be upcoming release as it will use RHEL repositories for pre-release, so at the moment 3.10). 

But if we chose to pass it for all, we might add logic to "load repos based on this variable". Because either we remove this, we will still need to maintain multiple branches for RPM channels (after release we will need to make PR to release 3.10 to change channels from pre-release to release).